lib/ome/qtwidgets/GLWindow.cpp

```diff
@@ -175,13 +176,12 @@ namespace ome
 
         glcontext = new QOpenGLContext(this);
         glcontext->setFormat(format);
-        glcontext->create();
+        bool valid = glcontext->create();
+        std::cerr << "Valid OpenGL context: " << valid << std::endl;
```

Comment:
For the moment, yes, while we're testing on different platforms.  One thing the code doesn't yet do is handle failure gracefully when it doesn't get a valid context.  It will segfault later on when we first try to use the invalid context.  But I see it return `true` even when apparently invalid, so it would be handy to have around until we make it behave properly under all conditions.  OpenGL setup and error handling is quite awful!